File: CursoJava/Exer/src/Outros/ListaAlgoritimo/Algoritimo3/Exer08.java
/*
8) Escreva um algoritmo que leia um valor inicial A e uma razão R e imprima uma seqüência em
P.A. contendo 10 valores.
 */

package Outros.ListaAlgoritimo.Algoritimo3;

public class Exer08 {
    public static void main(String[] args) {
        int a = 1;
        int r = 2;
        for (int i = 0; i < 10; i++) {
            System.out.println(a + (i * r));
        }
    }
}

File: CursoPython3/exer/lista_Algoritimos/algoritimo3/exer08.py
"""
8) Escreva um algoritmo que leia um valor inicial A e uma razão R e imprima uma seqüência em
P.A. contendo 10 valores.
"""

a = int(input("Digite o valor inicial: "))
r = int(input("Digite a razão: "))
for i in range(10):
    print(a)
    a += r
